projects/hslayers/src/components/query/partials/infopanel.html

```diff
@@ -1,5 +1,5 @@
 <hs-panel-header name="info" [title]="'PANEL_HEADER.INFO' | translate"></hs-panel-header>
-<div class="card hs-main-panel" *ngIf="HsQueryBaseService.data.coordinates === undefined || HsQueryBaseService.data.coordinates.length == 0">
+<div class="card hs-main-panel" *ngIf="HsQueryBaseService.data.features.length == 0">
```

Comment:
Need to double check if works with only WMS layers on map and no vector layers.

Reply:
@raitisbe Made a small adjustment